README.md

```diff
@@ -251,6 +251,68 @@ err := apiClient.UpdateUsersBatch(users)
 
 ```
 
+
+### Update company
+
+```go
+import "github.com/moesif/moesifapi-go"
+import "github.com/moesif/moesifapi-go/models"
+import "time"
+
+apiClient := moesifapi.NewAPI("my_application_id")
+
+modifiedTime := time.Now().UTC()
+
+metadata := map[string]interface{}{
+	"email": "johndoe@acmeinc.com",
+	"Key1": "metadata",
+	"Key2": 42,
+	"Key3": map[string]interface{}{
+		"Key3_1": "SomeValue",
+	},
+}
+
+company := models.CompanyModel{
+		ModifiedTime: 	  &modifiedTime,
```

Comment:
Let's use spaces here so the whitespace renders consistently

Reply:
Refactored to use spaces for consistency